internal/app/app.go

```diff
@@ -0,0 +1,71 @@
+package app
+
+import (
+	"log"
+	"os"
+	"os/signal"
+
+	"github.com/newUser1337/task-news/internal/config"
+	"github.com/newUser1337/task-news/internal/context"
+	"github.com/newUser1337/task-news/internal/migration"
+	"github.com/newUser1337/task-news/internal/repository"
+	"github.com/newUser1337/task-news/internal/router"
+	"github.com/newUser1337/task-news/internal/server"
+	"github.com/newUser1337/task-news/internal/service/news"
+	"github.com/newUser1337/task-news/internal/usecase"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func BuildApp() {
```

Comment:
it's a good practive to return error from a function if it produces errors. Then you can replace `log.Fatal` with `return err` and do `log.Fatal` in `main.go` instead.

Reply:
fixed